### PR TITLE
update documentation for managed plugin

### PR DESCRIPTION
**Pull request by raunakkumar**

### CODE REVIEW

File: cmd/dockervolumed/managedplugin/README.md [hunk 1]
```diff
@@ -76,51 +76,77 @@ Debian 9.x (stable):
...
 sed -i"" -e "\$axfs" /etc/modules
-docker plugin install --disable --grant-all-permissions --alias nimble store/nimblestorage/nimble:2.5.1
+docker plugin install --disable --grant-all-permissions --alias nimble store/hpestorage/nimble:3.0.0
 docker plugin set nimble PROVIDER_IP=192.168.171.74 PROVIDER_USERNAME=admin PROVIDER_PASSWORD=admin iscsiadm.source=/usr/bin/iscsiadm glibc_libs.source=/lib/x86_64-linux-gnu
 docker plugin enable nimble
 systemctl daemon-reload
 systemctl restart open-iscsi multipath-tools
 ```
 
+**NOTE:** To use the plugin on Fibre Channel environment use `PROTOCOL=FC` environment variable.
```
Comment (rgcostea) [inline cmd/dockervolumed/managedplugin/README.md]: nit: environments

File: cmd/dockervolumed/managedplugin/README.md [hunk 2]
```diff
@@ -76,51 +76,77 @@ Debian 9.x (stable):
...
 
-```$ docker plugin disable nimble
+```markdown
+
+$docker plugin disable nimble
 ```
 
+#### Settable Parameters
+
+List of parameters which are supportable to be settable by the plugin
```
Comment (rgcostea) [inline cmd/dockervolumed/managedplugin/README.md]: nit: "supportable to be settable" doesn't read right.

File: cmd/dockervolumed/managedplugin/README.md [hunk 3]
```diff
@@ -76,51 +76,77 @@ Debian 9.x (stable):
...
+
+|  Parameter              |  Description                                                |  Default |
+|-------------------------|-------------------------------------------------------------|----------|
+| `PROVIDER_IP`           | HPE Nimble Storage array IP                                 |`""`      |
+| `PROVIDER_USERNAME`     | HPE Nimble Storage array Username                           |`""`      |
+| `PROVIDER_PASSWORD`     | HPE Nimble Storage array password                           |`""`      |
```
Comment (rgcostea) [inline cmd/dockervolumed/managedplugin/README.md]: nit: why is Username capitalized and password is not?

File: cmd/dockervolumed/managedplugin/README.md (hunk 2 above)
Comment (raunakkumar) [reply cmd/dockervolumed/managedplugin/README.md]: will change to supported to be settable

File: cmd/dockervolumed/managedplugin/README.md (hunk 3 above)
Comment (raunakkumar) [reply cmd/dockervolumed/managedplugin/README.md]: will make it even :)

**Pull request merged**